### PR TITLE
deployment: notify mailing list on failures.

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

File: deploy/Jenkinsfile [hunk 1]
```diff
@@ -356,4 +356,16 @@ pipeline {
...
+    post {
+        changed {
+            script {
+                if (!env.ghprbPullId) {
+                    mail to: 'bbp-deployment@groupes.epfl.ch',
+                         subject: "Failed Deployment Build: ${currentBuild.fullDisplayName}",
+                         body: "See ${env.BUILD_URL}"
+                }
+            }
+        }
```
Comment (pramodk) [inline deploy/Jenkinsfile:369]: I assume this is only for nightly builds? Hope not too many emails :) Lets see!

**Review by pramodk — DISMISSED** [review]

File: deploy/Jenkinsfile (hunk 1 above)
Comment (matz-e) [reply deploy/Jenkinsfile:369]: Yes. Actually just noticed that I forgot a condition there / this should be for failed builds only

File: deploy/Jenkinsfile [hunk 2]
```diff
@@ -356,4 +356,16 @@ pipeline {
             }
         }
     }
+
+    post {
+        changed {
```
Comment (matz-e) [inline deploy/Jenkinsfile]:
```suggestion
        failure {
```

File: deploy/Jenkinsfile (hunk 1 above)
Comment (matz-e) [reply deploy/Jenkinsfile:369]: I think it's good to make this for every failed build, to really drive home when it fails continuously. The volume should still be much less than these permanent "issue has been moved to next sprint" notifications we receive.

**Comment by matz-e** [comment]
Ping!

File: deploy/Jenkinsfile (hunk 1 above)
Comment (alexsavulescu) [reply deploy/Jenkinsfile:369]: totally agree on the sprint notifications

**Review by alexsavulescu — APPROVED** [review]
LGTM

**Comment by matz-e** [comment]
> LGTM

BTW, I need to add you there :D

**Comment by alexsavulescu** [comment]
> > LGTM
> 
> BTW, I need to add you there :D

Nooooo, that's why I approved ! Just kidding ;)

**Comment by matz-e** [comment]
Well, I already got my first email, so it's working :dagger: 

(and it didn't fail with the addition)

**Pull request merged**